test_cwl.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+set -o errexit
+set -o pipefail
+
+red=`tput setaf 1`
+green=`tput setaf 2`
+reset=`tput sgr0`
+start() { echo travis_fold':'start:$1; echo "$green$1$reset"; }
+end() { set +v; echo travis_fold':'end:$1; echo; echo; }
+die() { set +v; echo "$red$*$reset" 1>&2 ; exit 1; }
+
+pushd .
```

Comment:
why? We're already at ".". (Maybe this just got copied over?)

Reply:
It did get copied over.